vispy/visuals/implicitmesh.py

```diff
@@ -0,0 +1,66 @@
+
+from .mesh import MeshVisual
+from ..geometry import create_implicit_mesh
+from ..color import ColorArray
+
+import numpy as np
+
+
+class ImplicitMeshVisual(MeshVisual):
```

Comment:
"implicit" is not descriptive enough, as there are many ways a mesh could be implicitly constructed around an array of vertices. I would recommend using "grid" instead--`GridMeshVisual`,  `create_grid_mesh()`, etc.